ACG-instructions.md

```diff
@@ -39,4 +39,8 @@ All of the commands you expect should work will work, and it also comes with a r
 ## Step 5 (Unit 1 Workshop): Install necessary tools for Unit 1 Workshop
 
 If you're doing the Unit 1 Workshop, you'll also need to ensure you install the AWS CLI.
-Run `sudo snap install aws-cli --classic` (you'll need your password)
+Run `sudo snap install aws-cli --classic` (you'll need your password).
```

Comment:
(As an alternative, we could recommend just using Amazon Linux 2 instead of Ubuntu, which comes with this pre-installed I believe)